mitodl/ocw-hugo-themes
# About Us Page

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/278

#### What's this PR do?
This PR adds the About Us page to the `ocw-www` theme, based on the designs in the attached issue.  It adds the milestone cards and OCW staff as new types of content, so they can be easily managed in the future and / or brought under CMS control.

#### How should this be manually tested?
 - Review the designs in the attached issue for mobile in desktop
 - View the PR [preview deploy of the page](https://ocw-hugo-themes-pr-303--ocw-next.netlify.app/about)
 - Verify that the look of the page is as close to the designs as possible
 - Try out the sub-nav on desktop and mobile, make sure it skips to the intended sections
 - Try out the milestone and staff carousels on mobile
 - Verify that links on the page go where they should
 - Verify that the embedded videos play
 - Repeat the above for as many browsers as you have access to, as well as at mobile resolutions

#### Screenshots (if appropriate)
![desktop-1](https://user-images.githubusercontent.com/12089658/147010441-9570b96a-56a3-4ca3-9143-39d91a69b58e.png)
![desktop-2](https://user-images.githubusercontent.com/12089658/147010460-e7279873-14d4-49e3-ad5e-d930cf6e46d3.png)
![desktop-3](https://user-images.githubusercontent.com/12089658/147010486-47be938e-2c2c-4ace-b98c-b31315f46d11.png)
![desktop-4](https://user-images.githubusercontent.com/12089658/147010532-91e9280b-f13b-4f75-bb5d-fc470a6e4775.png)
![desktop-5](https://user-images.githubusercontent.com/12089658/147010534-4ef087a2-c51c-4acd-a8cd-a8e6ce039962.png)
![desktop-6](https://user-images.githubusercontent.com/12089658/147010535-9a82af7b-6bd2-46c8-85b1-fff5ea22ca4c.png)
![desktop-7](https://user-images.githubusercontent.com/12089658/147010537-968daca6-e029-40bd-a817-507227776ea8.png)
![desktop-8](https://user-images.githubusercontent.com/12089658/147010538-f06aac3a-bcb3-45c4-b641-8dced9f74eab.png)
![desktop-9](https://user-images.githubusercontent.com/12089658/147010539-d939d6e7-bc11-4a51-b8ac-ef52e926ea52.png)
![desktop-10](https://user-images.githubusercontent.com/12089658/147010540-4fca282a-952c-439e-bc63-dc814ae568bd.png)
![mobile-1](https://user-images.githubusercontent.com/12089658/147010541-29030af8-2d64-474c-b4b7-643fa0b57ad3.png)
![mobile-2](https://user-images.githubusercontent.com/12089658/147010542-cc0732ec-aa57-42d9-9662-76bc6d0e8d0e.png)
![mobile-3](https://user-images.githubusercontent.com/12089658/147010544-821b3cfe-30eb-4b62-8800-fb4834beb3d2.png)
![mobile-4](https://user-images.githubusercontent.com/12089658/147010546-cf9bae01-b6d0-4ba7-a22e-ea836681372c.png)
![mobile-5](https://user-images.githubusercontent.com/12089658/147011728-068f7728-9ee5-4af8-8627-8d389ff2305d.png)
![mobile-6](https://user-images.githubusercontent.com/12089658/147010548-c1779fe7-5f3f-4fe2-8bdd-3a78d96ee4fb.png)
